model/distro/distro.go

```diff
@@ -41,7 +41,7 @@ type Distro struct {
 // BootstrapSettings encapsulates all settings related to bootstrapping hosts.
 type BootstrapSettings struct {
 	Method                string `bson:"method" json:"method" mapstructure:"method"`
-	Communication         string `bson:"communcation,omitempty" json:"communication,omitempty" mapstructure:"communcation,omitempty"`
+	Communication         string `bson:"communication,omitempty" json:"communication,omitempty" mapstructure:"communication,omitempty"`
```

Comment:
Have these already been persisted to the db? If so, you cannot change tags without a migration.

Reply:
Answered in person - it's persisted but technically it doesn't matter since everything runs on legacy SSH currently (and the default behavior is legacy SSH if none is given).